source/install/cloud-changelog.md

```diff
@@ -3,12 +3,38 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-12-01](#release-2021-12-01)
 - [Release 2021-11-23](#release-2021-11-23)
 - [Release 2021-11-11](#release-2021-11-11)
 - [Release 2021-11-10](#release-2021-11-10)
 - [Release 2021-10-27](#release-2021-10-27)
 - [Release 2021-10-13](#release-2021-10-13)
-- [Release 2021-09-29](#release-2021-09-29)
+
+## Release 2021-12-01
+
+### Improvements
+
+#### User Interface (UI)
+ - Clarified Latex Rendering config setting descriptions and fixed a broken product documentation link.
+ - Updated the "One-click reactions on messages" user setting to "Quick reactions on messages".
+ - Added tab focus support to the global header and user avatars.
+ - Added a new **Replies** banner to the right-hand side Thread viewer.
+
+### Bug Fixes
+ - Fixed an issue where Mattermost Cloud installations were panicing after the November 23, 2021 release.
+ - Fixed an issue where the default log rotation file size was mistakenly set to 10GB, and is now reverted back to 100MB.
+ - Fixed an issue where OpenID redirects didn't work when hosting Mattermost on a subdirectory.
+ - Fixed an issue where using CMD/CTRL + SHIFT + F in global threads did not add a search term automatically.
+ - Fixed the alignment of the “X” button in the “message deleted” System message.
+ - Fixed an issue where the long post "Show More/Less" background was broken in the Thread viewer.
+
+### Known Issues
+ - Member type is missing from autocomplete [MM-38989](https://mattermost.atlassian.net/browse/MM-38989).
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CMD+/`` does not close the shortcuts modal [MM-38971](https://mattermost.atlassian.net/browse/MM-38971).
+ - ``CTRL/CMD + SHIFT + A`` shortcut does not open **Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
+ - Known issues related to the new Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - System Roles shows **License** and **Environment** as possible permissions, but they are always hidden in Cloud.
```

Comment:
@amyblais just curious about this, is there a bug for this? Would like to take a look

Reply:
@marianunez Ticket: https://mattermost.atlassian.net/browse/MM-30941. It has been on the list for a while, so I can remove it from here.

Reply:
0/5 if keeping or not, I just wanted to see the ticket so I can try to prioritize the fix for it.